modes/usAnnotation/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@ohif/mode-usAnnotation",
```

Comment:
This doesn't follow our convention; it should be `@ohif/mode-ultrasound-bline-plural` or something similar.